mobileapp/android/app/src/main/AndroidManifest.xml

```diff
@@ -15,7 +15,7 @@
             android:theme="@style/AppTheme.NoActionBarLaunch"
             android:launchMode="singleTask"
             android:exported="true"
-            android:screenOrientation="portrait">
+            android:screenOrientation="unspecified">
```

Comment:
If this is unspecified can't we just remove it or is it still a requirement for capacitor/the APK?

Reply:
yeah it is the default so we can remove it, little more explicit and makes it a little easier for searching later if it's there. Can remove if you think it's nicer ✂️

Reply:
I don't think it matters much but I guess removing it could have unintended consequences if the default change?